File: tests/csharp/IndirectExec.cs
using System;
using System.Diagnostics;

class IndirectExec {
    static void Main(string[] args) {
        string cmd = args[0];
        // Indirect call: assign Process.Start to Action
        Action<string> func = (c) => {
            Process.Start(new ProcessStartInfo(c) { UseShellExecute = true });
        };
        func(cmd);
    }
}

File: tests/java/IndirectExec.java
import java.io.*;

public class IndirectExec {
    // Indirect call: Runnable wraps Runtime.exec
    public static void main(String[] args) throws Exception {
        String cmd = args[0];
        // Indirect call via lambda
        Runnable r = () -> {
            try { Runtime.getRuntime().exec(cmd); } catch (Exception e) {}
        };
        r.run();
    }
}
